webapp/playbooks/babel.config.js

```diff
@@ -48,19 +48,28 @@ const config = {
     ],
 };
 
-const NPM_TARGET = process.env.npm_lifecycle_event; //eslint-disable-line no-process-env
-const targetIsDevServer = NPM_TARGET === 'dev-server';
-if (targetIsDevServer) {
-    config.plugins.push(require.resolve('react-refresh/babel'));
-}
```

Comment:
Because we remove dev-server from playbooks https://github.com/mattermost/mattermost-server/pull/23378/files#diff-c1548712bd2d9f8c157507803c42c2e3ce966ac39766eda5b2c8d9c3962fa258

Reply:
Whoops, I thought I'd gotten rid of that. I must've reverted it when I removed some other half-finished changes